src/axis.js

```diff
@@ -1,4 +1,14 @@
 import { timeFormat } from 'd3-time-format';
+import breakpoints from './breakpoints';
+
+export const getBreakpointLabel = point =>
+    Object.keys(breakpoints).reduce((accumulator, label) => {
```

Comment:
Why not just using a `for` loop? It would prevent browsing every breakpoints for smallest devices.